homework/index.js

```diff
@@ -27,21 +27,163 @@
         elem.setAttribute(key, value);
       }
     });
+
     return elem;
   }
 
+  function renderRepositoryInfo(repository, leftContainer) {
+    const table = createAndAppend('table', leftContainer);
+    const tbody = createAndAppend('tbody', table);
+    const firstRow = createAndAppend('tr', tbody, {
+      tabindex: '0',
+    });
+    createAndAppend('td', firstRow, {
+      class: 'label',
+      text: 'Repository : ',
+    });
+    const td2 = createAndAppend('td', firstRow);
+    createAndAppend('a', td2, {
+      text: repository.name,
+      href: repository.html_url,
+    });
+
+    const secondRow = createAndAppend('tr', tbody, {
+      tabindex: '0',
+    });
+    createAndAppend('td', secondRow, {
+      class: 'label',
+      text: 'Description :',
+    });
+    createAndAppend('td', secondRow, {
+      text: repository.description,
+    });
+    const thirdRow = createAndAppend('tr', tbody, {
+      tabindex: '0',
+    });
+    createAndAppend('td', thirdRow, {
+      class: 'label',
+      text: 'Forks :',
+    });
+    createAndAppend('td', thirdRow, {
+      text: repository.forks,
+    });
+    const forthRow = createAndAppend('tr', tbody, {
+      tabindex: '0',
+    });
+    createAndAppend('td', forthRow, {
+      class: 'label',
+      text: 'Updated :',
+    });
+    createAndAppend('td', forthRow, {
+      text: repository.updated_at,
```

Comment:
The format of the displayed date could be improved, for instance by using this method on `new Date(repository.updated_at)`: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date/toLocaleDateString